
# mav offb control type for diff tasks
uint8 mav_offb_ctl_type

uint8 OFFBOARD_IGNORE = 0

uint8 GOTO_SETPOINT_STEP = 1
uint8 GOTO_SETPOINT_SMOOTH = 2

uint8 GOTO_SETPOINTS_STEP = 3
uint8 GOTO_SETPOINTS_SMOOTH = 4

uint8 POS_CTRL_BY_ATT = 5

uint8 POS_CTRL_BY_BODY_RATE = 6